webapp/channels/src/components/widgets/icons/fa_previous_icon.tsx

```diff
@@ -8,21 +8,18 @@ import LocalizedIcon from 'components/localized_icon';
 import {t} from 'utils/i18n';
 
 type Props = {
-    additionalClassName: string | null;
+    additionalClassName?: string;
 }
 
-export default class PreviousIcon extends React.PureComponent<Props> {
-    public static defaultProps: Props = {
-        additionalClassName: null,
-    };
-
-    public render(): JSX.Element {
-        const className = 'icon icon-chevron-left' + (this.props.additionalClassName ? ' ' + this.props.additionalClassName : '');
-        return (
-            <LocalizedIcon
-                className={className}
-                title={{id: t('generic_icons.previous'), defaultMessage: 'Previous Icon'}}
-            />
-        );
-    }
+function PreviousIcon(props: Props) {
```

Comment:
Exactly the same 3 comments I gave you on https://github.com/mattermost/mattermost/pull/24787